api/v1alpha1/contour_types.go

```diff
@@ -302,6 +314,40 @@ const (
 	GCPLoadBalancerProvider   LoadBalancerProviderType = "GCP"
 )
 
+type AwsLoadBalancerParameters struct {
+	// eip is the id of created Elastic IP
```

Comment:
@danehans Thank you again for talking all this time to help me with this.
* Periods added to end of all sentences.
* I added example in the comment. I adjusted this structure a bit, so users can provide multiple Elastic IPs/their allocation IDs
* Doc links added
* You are correct, this is Allocation ID of elastic IP. I also changed type from `int` to `[]string` so that users can now input multiple allocation IDs
* Name changed from `eid` to `allocationId`
* All sentence beginnings were capitalized